frontend/src/v5/ui/components/viewer/drawingViewer/drawingViewer.component.tsx

```diff
@@ -0,0 +1,113 @@
+/**
```

Comment:
**BUG**
(this only happens with pngs, not svgs)
After zooming, once, it is possible to drag the zoom icon and force the image into a dragging state
https://github.com/3drepo/3drepo.io/assets/43547703/56cf8a19-6c71-4f1e-afe6-55b8fa203947